packages/algob/src/lib/script-checkpoints.ts

```diff
@@ -223,9 +223,9 @@ export async function registerCheckpoints (
           const asaInfo: rtypes.ASAInfo = {
             creator: encodeAddress(txn.from.publicKey),
             txId: txn.txID(),
-            assetIndex: txConfirmation["asset-index"],
-            confirmedRound: txConfirmation['confirmed-round'],
-            assetDef: res[1],
+            assetIndex: Number(txConfirmation.assetIndex),
```

Comment:
Are we doing `Number(assetIndex)` because algosdk type is `number | bigint` ? If yes then can we update `rtypes.ASAInfo.assetIIndex` to `number | bigint` (same as SDK) ?

Reply:
the main problem is undefined, it is a compiler issue. We can change it to bigInt or number